### ISSUE TITLE
Problems with positioning paneContainer absolutely

### ISSUE THREAD

**Issue by victor-ponamariov** [imgix/drift#80]
Hello!

I searched both open and closed issues, probably missed something.

I have a pane container which is positioned absolutely. This container is placed on a div which contains some information about a product, some buttons and other stuff. I need to be able to show the pane container when user hovers a picture, and then hide the container on mouse out.

The problem is, that, I tried to use onShow and onHide events, but in that case bounding-box does not have width and height, and thus I cannot see it at all. I think this happens because when the pane is hidden for some reason bounding-box does not get width/height css attributes.

More details: I use VueJS, and I have <div class="pane-container" v-show="showPaneContainer">. I change showPaneContainer in onHide/onShow callbacks.

So basically bounding-box works fine only if the paneContainer is always visible, but in that case user cannot click on any buttons or copy text or do something else when the paneContainer **covers** underline div with product info.

I cannot use existing div with product info as pane container because in VueJS these are two different separate components and it'll be inconvenient to do that.

**Comment by frederickfogerty** [comment]
Hey there @victor-ponamariov, thanks for the bug report. This seems like a pretty in depth issue so I'm going to have to spend some time reproducing. I hope to do this later this week or early next week.

**Comment by frederickfogerty** [comment]
Hi there @victor-ponamariov 👋 

If I've understood your issue correctly, it's that the paneContainer doesn't exist before Drift measures the width, and thus `getBoundingClientRect()` fails.

I've adapted the classic Drift example on Codepen to work as you've described. Please let me know if this solves your issue. In the example, `.paneContainer` is absolutely positioned and is equivalent to your pane.

**Comment by frederickfogerty** [comment]
Hi @victor-ponamariov, I'm going to close this now as it's been inactive for a few days! If you have any more issues please comment on this issue or create a new issue. Thanks and have a great day! 👋
